zenoh/src/api/session.rs

```diff
@@ -1933,17 +1932,14 @@ impl SessionInner {
                 .subscribers(SubscriberKind::Subscriber)
                 .values()
                 .any(|s| s.key_expr.intersects(key_expr)),
-            MatchingStatusType::Queryables(false) => state.queryables.values().any(|q| {
-                state
-                    .local_wireexpr_to_expr(&q.key_expr)
-                    .is_ok_and(|ke| ke.intersects(key_expr))
-            }),
-            MatchingStatusType::Queryables(true) => state.queryables.values().any(|q| {
-                q.complete
-                    && state
-                        .local_wireexpr_to_expr(&q.key_expr)
-                        .is_ok_and(|ke| ke.includes(key_expr))
-            }),
+            MatchingStatusType::Queryables(false) => state
+                .queryables
+                .values()
+                .any(|q| q.key_expr.includes(key_expr)),
```

Comment:
```suggestion
                .any(|q| q.key_expr.intersects(key_expr)),
```

Must've been a typo.

On an unrelated note, I couldn't find any documentation of how complete query/reply uses _include_ and not _intersect_, is this documented somewhere?